src/components/MasterItemsSelect.js

```diff
@@ -1,25 +1,61 @@
 import { Component, PropTypes, DOM as dom } from 'react';
+import _ from 'underscore';
 
 class MasterItemsSelect extends Component {
   constructor(props) {
     super(props);
     this.handleChange = this.handleChange.bind(this);
+    this.handleChangeCategory = this.handleChangeCategory.bind(this);
+    this.state = {
+      filteredItems: this.filterItems(this.props.masterItems[0].category)
```

Comment:
Any thoughts on this? Had to do this to get the first category and filter it, otherwise, it would show everything item.